src/modules/webpack-publish.test.ts

```diff
@@ -0,0 +1,123 @@
+import { WebpackRestoreOptions } from "../interfaces/webpack-restore-options";
+import { NodeCI } from "./node-ci";
+import { NodeClean } from "./node-clean";
+import { NodeRestore } from "./node-restore";
+import { WebpackPublish } from "./webpack-publish";
+
+describe("webpack-publish", () => {
+    describe("restore", () => {
+        beforeEach(() => {
+            jest.spyOn(NodeClean, "run").mockImplementation();
+            jest.spyOn(NodeClean, "run").mockImplementation();
+            jest.spyOn(NodeCI, "run").mockImplementation();
+        });
+        test("when ci is null, then NodeCI is not called ", () => {
+            // Arrange
+            const nodeCISpy = jest.spyOn(NodeCI, "run");
+
+            const options: WebpackRestoreOptions = {
+                ci: undefined,
+                skipClean: undefined,
+                skipRestore: undefined,
+            };
+
+            // Act
+
+            WebpackPublish.restore(options);
+
+            // Assert
+            expect(nodeCISpy).not.toBeCalled();
+        });
+        test("when ci is false, then NodeCI is not called ", () => {
+            // Arrange
+            const options: WebpackRestoreOptions = {
+                ci: false,
+                skipClean: undefined,
+                skipRestore: undefined,
+            };
+
+            // Act
+            const nodeCISpy = jest.spyOn(NodeCI, "run");
```

Comment:
Recommend moving spy setup into the 'arrange' above as you have it in previous tests